nexus/src/external_api/http_entrypoints.rs

```diff
@@ -3327,6 +3333,129 @@ async fn instance_disk_detach(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+// Certificates
+
+/// List system-wide certificates
+///
+/// Returns a list of all the system-wide certificates. System-wide certificates are returned sorted
```

Comment:
From the user's perspective, these are more specifically "TLS certificates for the externally-facing API servers", right?

I wonder if we even want to put that into the URL.  In the future we might manage other kinds of certificates and I imagine we'd want separate endpoints to manage them.

Reply:
I'm using a service type in the input and output: fc877e330f7b978265d91996faa596b228e8f786 . It's only nexus now, but hopefully this makes future certificate usage more clear